pitch.md

```diff
@@ -23,38 +37,125 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
+The programmer should also have a way to draw a check mark at the end of each line of the tree.
+Tableau trees are binary trees so the programmer can also specify if the tree branches off.
+
+One possible syntax is
+```
+(A \or B) \and C \check
+C
+A \or B
+  A
+    branch1
+    branch2
+  B
+```
+to draw
+
+![tree](/images/tree.png)
+
+which is similar to how qtree, a LaTeX package for trees, structures code but this doesn't require entering math mode using \$.
+Anthony mentioned in his critique that the backslash may not be necessary in this language.
+This would restrict programmers from having variable names like 'or' and 'and', but then again those would be bad variable names in a logic proof.
 
 ## Operation (semantics)
 _What might happen when a program runs? Is there a computational model that
 corresponds with your domain? How does a program interact with the
 user? What kinds of semantic (i.e., non-syntax) errors might occur, and how 
 might they be communicated to the user?_
 
+The file is read an compiled.
+A program would render a tree based on the text file and output an image file.
+This fits the data-entry model.
+Drawing a tree is a sub-problem of drawing a graph, which was the given example for the data-entry model.
+
+Syntax errors, such as branching in 3 directions, should be reported after compilation, before the tree is drawn.
+This could be reported by writing to an error stream or file.
+
+Semantic errors would result in a malformed tree.
+I'm not sure how this would be reported to the user other than by having an image file of a weird tree.
 
 ## Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to draw any binary tree with logic symbols.
+
+It should be possible, but maybe not necessarily easy, to draw trees with text that contains symbols other than logic symbols.
+Maybe Unicode symbols.
+It should be possible to use different kinds of logic symbols.
+For example, not can be represented with ¬ or !, the user should be able to choose which set of symbols they want to use.
+
+It should be impossible to draw trees that are not binary.
+It should also be impossible to draw a graph with a cycle.
 
 ## Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+There are LaTeX packages that logicians use for drawing trees.
+[Here](http://www.logicmatters.net/latex-for-logicians/trees/) are some examples.
+However, installing LaTeX is not very easy as the common TeX distributions are quite large.
+LaTeX also has a steep learning curve, which may not be worth it for someone who only needs to draw logic trees.
```

Comment:
As someone who knows very little about tableau trees, the idea of being able to use your DSL instead of having to download and learn an entire LaTeX package is very appealing!